<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextReader</class>
 <widget class="QWidget" name="TextReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>语言选择：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'宋体,simsun,sans-serif,Arial'; font-size:14px; color:#000000; background-color:#ffffff;&quot;&gt;2016年1月14日周四&lt;/span&gt;&lt;a href=&quot;http://q.stock.sohu.com/zs/000001/index.shtml&quot;&gt;&lt;span style=&quot; font-family:'宋体,simsun,sans-serif,Arial'; font-size:14px; text-decoration: underline; color:#005599;&quot;&gt;沪指&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'宋体,simsun,sans-serif,Arial'; font-size:14px; color:#000000;&quot;&gt;低开高走全天震荡上行收盘重回3000点，全天涨幅1.97%，两市成交量突破6000亿，有所回升。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'宋体,simsun,sans-serif,Arial'; font-size:14px; color:#000000; background-color:#ffffff;&quot;&gt;同日，余额宝情绪指数1699点，较上一个交易日大幅上升，投资者信心有所恢复。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'宋体,simsun,sans-serif,Arial'; font-size:14px; color:#000000; background-color:#ffffff;&quot;&gt;由于人民币快速贬值在近两天得到一定缓解，再加上已经公布的12月宏观数据高于预期等原因，昨日A股反弹上涨。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'宋体,simsun,sans-serif,Arial'; font-size:14px; color:#000000; background-color:#ffffff;&quot;&gt;然而，引发近期股市暴跌的因素仍然存在，市场风险尚未消除。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>播放</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
